static/js/filter.js

```diff
@@ -236,8 +236,9 @@ Filter.parse = function (str) {
             // it as a search for the given string (which may contain
             // a `:`), not as a search operator.
             if (Filter.operator_to_prefix(operator, negated) === '') {
-                operator = 'search';
-                operand = token;
+                // Put it as a search term, to not have duplicate operators
+                search_term.push(token);
+                return;
```

Comment:
Can you add a Node test for this case in frontend_tests/node_tests/filter.js?  We've been trying to keep that a comprehensive test suite for this filter logic since it's complicated, and I think this is a bug worth having a test for there.